src/UserBundle/Resources/skeleton/security.php

```diff
@@ -31,7 +31,7 @@
                 default_target_path: /profile
                 username_parameter: ${fieldName}
                 password_parameter: password
-
+${logout}
```

Comment:
you can declare `$logoutSection` based on `if ($logout) ...` which we assigned above, that avoids the extra file